From af2ff55739187c59dc04e396b7787301a49a9dba Mon Sep 17 00:00:00 2001
From: Michael Calmer <mc@suse.de>
Date: Fri, 29 Jul 2016 10:50:21 +0200
Subject: [PATCH 12/12] Run salt-api as user salt (bsc#990029)

---
 pkg/salt-api.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pkg/salt-api.service b/pkg/salt-api.service
index 72379ba..9be2cb8 100644
--- a/pkg/salt-api.service
+++ b/pkg/salt-api.service
@@ -3,6 +3,7 @@ Description=The Salt API
 After=network.target
 
 [Service]
+User=salt
 Type=simple
 LimitNOFILE=8192
 ExecStart=/usr/bin/salt-api
-- 
2.9.2

